sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_custom_documents.py

```diff
@@ -89,6 +89,22 @@ def analyze_custom_documents(custom_model_id):
                     print(f"...{region.page_number}")
             for cell in table.cells:
                 print(f"...Cell[{cell.row_index}][{cell.column_index}] has text '{cell.content}'")
+
+    # Extract table cell values from "documents" field in result and print the name/value information
+    if result.documents:
+        for idx, doc in enumerate(result.documents):
+            print(f"\n{doc.fields.__len__()} fields found on Document {idx}" )
+            for fieldName,fieldValue in doc.fields.items():
+                # "TableLayout" is the table field name which you labeled. Table cell information store as array in documnet field.
+                if fieldName=="TableLayout" and fieldValue.type=="array" and fieldValue.value_array:
```

Comment:
nit: you need to spare whitespaces before and after `==`. can you enable python auto formatter in vscode and use it to format your code?